hack/run-functests.sh

```diff
@@ -10,6 +10,8 @@ if [ $? -ne 0 ]; then
 fi
 
 GOPATH="${GOPATH:-~/go}"
+
+export IMAGE_RUN_CMD="${IMAGE_RUN_CMD:-docker}"
```

Comment:
CONTAINER_MGMT_CLI ?
image_run_cmd is only understandable when you see the value :-)

Reply:
It's in the same line, so I don't think is too obscure. Will change though.

Reply:
it might be, further down the line..